src/main/java/seedu/address/model/InterviewSchedule.java

```diff
@@ -0,0 +1,103 @@
+package seedu.address.model;
```

Comment:
Looks good! I believe this file is similar to `src/main/java/seedu/address/model/AddressBook.java`. Glad there's a reference! Well thought out leon!